plugins/experimental/url_sig/url_sig.c

```diff
@@ -16,6 +16,13 @@
   limitations under the License.
  */
 
+#define min(a, b)           \
```

Comment:
Clever, but at this point, wouldn't it be easier / better to just make an inline min() function?